### PR TITLE
Project description

### PR DESCRIPTION
**Pull request by Eghizio**
We are adding some `basic description` of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
 # Rock Paper Scissors - Git
 
-Some description.
+We are learning **git** & **GitHub**.
+
+`hello from vim`
+
+> :wq
```
Comment (Eghizio) [inline README.md:7]: Oh great. I've always wanted to escape from `vim`!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+## Topics:
```
Comment (Eghizio) [inline README.md:9]: This list could be converted to a table with checkmarks.
This way we could track the progress of our study process.

**Review by Eghizio — COMMENTED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (Dominik-Maj) [reply README.md:7]: > Oh great. I've always wanted to escape from `vim`!

🥇

**Review by Dominik-Maj — APPROVED** [review]

**Pull request merged**